webview/src/test/nodes.ts

```diff
@@ -0,0 +1,3 @@
+export const idToNodeNode = (id: string) =>
```

Comment:
Is this supposed to be `idToNode`?

Reply:
Thanks. I renamed it and must have not selected the whole thing.